.github/workflows/build-sanity.yaml

```
@@ -8,6 +8,7 @@ on:
 env:
   MAVEN_OPTS: "-Dhttps.protocols=TLSv1.2 -Dmaven.color=false"
   MVN_CLI_ARGS: "--batch-mode --no-transfer-progress --fail-at-end --show-version"
+  MVN_SKIP_ARGS: "-Dformatter.skip -Denforcer.skip -Djacoco.skip -Dimpsort.skip -Dmdep.analyze.skip"
```

Please use our grouping property `skipFormatting` (which will disable the formatter plugin, the impsort plugin, and the pom sort plugin altogether).

```suggestion
  MVN_SKIP_ARGS: "-DskipFormatting -Denforcer.skip -Djacoco.skip -Dmdep.analyze.skip"
```